resources/metadata/newmanExecute.yaml

```diff
@@ -25,7 +25,7 @@ spec:
           - STEPS
         type: string
       - name: runOptions
-        description: The newman command that will be executed inside the docker container.
+        description: The newman command that will be executed inside the docker container. Env vars can be passed via template as in "{{getenv MY_ENV_VAR}}".
```

Comment:
Looks like the newmanExecute.yaml is also run through a template processor and which then tries to interpret the placeholder in the comment here. Googling tells me that you can probably solve this by writing it like this:
```
{{`{{getenv MY_ENV_VAR}}`}}
```

Reply:
Thanks for googling that for me.

Reply:
Seems like I was wrong though, since the error is still there? Sorry about that!  
I don't know enough about what is going on in the documentation build.

Reply:
I will simply remove this part of the documentation. We have it in the md file instead

Reply:
Oh, it appears it was the `documentation/docs/steps/newmanExecute.md` that it complained about 🙈

Reply:
Fixed it with [40e74b9](https://github.com/SAP/jenkins-library/pull/3966/commits/40e74b928c37012a6c8eb1b284d5b926c537d94b) but I'm unsure how it will look in the docs later. Need to check that after merge.

Reply:
Now the docu generation fails again, after merging master into here... :(